README.md

```diff
@@ -1,2 +1,201 @@
+
+
+- **Incorporate React components.**
+- **Implement React props.**
+- **Integrate React Router for navigation.**
+- **Establish a connection between React and Redux.**
+- **Manage events within a React application.**
+- **Compose integration tests using a React testing library.**
+- **Apply styles in a React application.**
+- **Employ React lifecycle methods effectively.**
+- **Adhere to React best practices and language style guides in your code.**
+- **Utilize store, actions, and reducers in React.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To clone and run this application, you'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer.
+
+### Prerequisites
+From your command line:
+
+On macOS, by using [Homebrew](https://brew.sh/) you can install.
+
+```bash
+# install git
+$ brew install git
+```
+
+```bash
+# install node
+$ brew install node
+```
+
+On Windows, by using [chocolatey](https://chocolatey.org) you can install.
+
+```bash
+# install git
+C:\> choco install git
+```
+
+
+```bash
+# install node
+C:\> choco install node
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+# Clone this repository
+$ git clone https://github.com/yemidada/bootstore.git
+
+# Go into the repository
+$ cd bootstore
+
+# Install dependencies
+$ npm install
+
+```
+
+### Usage
+
+After you clone this repo to your desktop and install the dependencies, go to its root directory and run `npm start` to run the application.
+
+### Live Demo and Video
+
+- [Live Demo Link](https://hairstyles.onrender.com/)
+- [Video Walkthrough](https://www.loom.com/share/9448458b6ebd4cf4992b6c8dd9da543c)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@yemidada](https://github.com/yemidada)
+- Twitter: [@naijavine](https://twitter.com/naijavine)
+- LinkedIn: [yemidada](https://linkedin.com/in/yemidada)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Improve the designs**
+- [ ] **Add more features**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can support us  by sharing our code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks you microverse for the opinion to opportunity and <a target="_blank" href="https://ffxivcollect.com">https://ffxivcollect.com</a> for the API
```

Comment:
- [ ] It would be nice to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) for the design inspiration used for this project. Kindly add that 👍🏽